cli/formatter/container.go

```diff
@@ -0,0 +1,49 @@
+package formatter
+
+import (
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/pkg/utils"
+)
+
+// ContainerHeader is the map to show container head
+var ContainerHeader = map[string]string{
+	"Name":    "Name",
+	"ID":      "ID",
+	"Status":  "Status",
+	"Created": "Created",
+	"Image":   "Image",
+	"Runtime": "Runtime",
+	"Command": "Command",
+	"ImageID": "ImageID",
+	"Labels":  "Labels",
+	"Mounts":  "Mounts",
+	"State":   "State",
+}
+
+// ContainerContext is the map to show container context detail
+type ContainerContext map[string]string
+
+// NewContainerContext is to generate a ContainerContext to be show
+func NewContainerContext(c *types.Container, flagNoTrunc bool) ContainerContext {
+	id := c.ID[:6]
+	if flagNoTrunc {
+		id = c.ID
+	}
+	created, _ := utils.FormatTimeInterval(c.Created)
```

Comment:
could we check the error here? I think the original code follows the error check here.

Reply:
done